### DISCUSSION TITLE
How do I pass `--remote-write.config-file` setting to ThanosRuler

### DISCUSSION THREAD

**Discussion by jihuiyang** [prometheus-operator/prometheus-operator#5439] [Q&A] [answered]
Hi, I'm trying to set up ThanosRuler to remotely write to another cluster. https://github.com/thanos-io/thanos/blob/main/docs/components/rule.md#stateless-ruler-via-remote-write. Does prometheus-operator support that? If so how do I configure it? Thanks!

**Comment by simonpasquier** [comment, marked as answer]
see #4451

**Reply by jihuiyang** [reply]
Any idea on when this will become available?

**Reply by jihuiyang** [reply]
So I removed `kind: ThanosRuler` [definition](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/thanos.md#thanos-ruler) from my config yaml, and created my own thanos-ruler config yaml. But when I apply it to k8s, in addition to the thanos-ruler resource I provisioned, prometheus-operator is still creating thanos-ruler resources: `service/thanos-ruler-operated`, `statefulset.apps/thanos-ruler-thanos-ruler`. Do you know how I can disable `prometheus-operator` from creating these redundant resources?
